doc/use.rst

```diff
@@ -0,0 +1,59 @@
+:orphan:
+
+Using MNE-BIDS
+==============
+
+.. contents:: Contents
+   :local:
+   :depth: 1
+
+Python API reference
+--------------------
+See the `API documentation <api>`_.
+
+Command line interface reference
+--------------------------------
+See the `CLI documentation <generated/cli>`_.
+
+Quickstart
+----------
+MNE-BIDS fully supports writing of BIDS datasets for MEG and EEG. Support for
+iEEG is experimental at the moment.
```

Comment:
is iEEG support still experimental? I feel like there have been many steps towards full ieeg functionality by @alexrockhill @adam2392 et al

Reply:
I've removed this line in 0a5af78